roles/install-calico/tasks/main.yml

```diff
@@ -1,8 +1,9 @@
 - name: Download calico manifest
-  get_url:
-    url: https://docs.projectcalico.org/manifests/calico.yaml
-    dest: /tmp/calico.yaml
-    mode: '0755'
+  shell: |
+    wget --limit-rate 1k https://docs.projectcalico.org/manifests/calico.yaml -P /tmp
+    chmod 0755 /tmp/calico.yaml
```

Comment:
is it required to clear existing `calico.yaml` if there any ?

Reply:
Usually, we have been using this on a new VM hence there wasn't existing calico.yaml.

But yes it shall would be correct to change it to `wget --limit-rate 1k https://docs.projectcalico.org/manifests/calico.yaml -O /tmp/calico.yaml` instaed of the -P option to `wget`